shell.nix

```diff
@@ -0,0 +1,51 @@
+{
+  sources ? import ./npins,
+  lib ? pkgs.lib,
+  pkgs ? import sources.nixpkgs {
+    config.android_sdk.accept_license = true;
+    config.allowUnfreePredicate =
+      pkg:
+      builtins.elem (lib.getName pkg) [
+        "android-sdk-platform-tools"
+        "android-sdk-cmdline-tools"
+        "android-sdk-build-tools"
+        "android-sdk-platforms"
+        "android-sdk-tools"
+
+        "android-sdk-ndk"
+        "android-sdk-ndk-bundle"
+      ];
+  },
+}:
+let
+  # pkgs.androidenv.androidPkgs.androidsdk pulls in lots of toolchains, slim it down
+  android = pkgs.androidenv.composeAndroidPackages {
+    includeNDK = true;
+    ndkVersions = [
+      "22.1.7171670"
+      "23.1.7779620"
+    ];
+    platformVersions = [
+      "28"
+      "30"
+    ];
+    buildToolsVersions = [ "30.0.3" ];
+    includeEmulator = false;
+    includeSystemImages = false;
+  };
+  jdk = pkgs.jdk11_headless;
```

Comment:
I thought `JavaVersion.VERSION_1_8` means jdk8 but that didn't work so using jdk11